playwright.config.ts

```diff
@@ -54,12 +54,21 @@ export default defineConfig<PluginOptions>({
     // Run all tests in parallel using user with viewer role
     {
       name: 'viewer',
-      testDir: path.join(testDirRoot, '/as-viewer-user'),
+      testDir: path.join(testDirRoot, '/plugin-e2e-api-tests/as-viewer-user'),
       use: {
         ...devices['Desktop Chrome'],
         storageState: 'playwright/.auth/viewer.json',
       },
       dependencies: ['createUserAndAuthenticate'],
     },
+    {
+      name: 'various',
+      testDir: path.join(testDirRoot, '/various-suite'),
```

Comment:
Change name to `mysql` and point to `mysql` testdir.

Reply:
done